<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="mainWidget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="GeometryView" name="graphicsView">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuElements">
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Elements</string>
    </property>
    <addaction name="actionPunkt"/>
    <addaction name="actionStrecke"/>
    <addaction name="actionKreis"/>
    <addaction name="actionVisible"/>
    <addaction name="actionRemove"/>
    <addaction name="actionCleanTracker"/>
   </widget>
   <addaction name="menuElements"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionSelect"/>
   <addaction name="actionPunkt"/>
   <addaction name="actionMidPoint"/>
   <addaction name="actionStrecke"/>
   <addaction name="actionKreis"/>
   <addaction name="actionLine"/>
   <addaction name="actionPointAtCircle"/>
   <addaction name="actionCircleIntersection"/>
   <addaction name="actionDynamicCircle"/>
   <addaction name="actionTracker"/>
  </widget>
  <action name="actionPunkt">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Point</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionStrecke">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Segment</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionKreis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetWithChildrenShortcut</enum>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionPointAtCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PointAtCircle</string>
   </property>
  </action>
  <action name="actionDynamicCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DynamicCircle</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionVisible">
   <property name="text">
    <string>Visible</string>
   </property>
  </action>
  <action name="actionTracker">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tracker</string>
   </property>
  </action>
  <action name="actionCleanTracker">
   <property name="text">
    <string>CleanTracker</string>
   </property>
  </action>
  <action name="actionMidPoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MidPoint</string>
   </property>
  </action>
  <action name="actionCircleIntersection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CircleIntersection</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GeometryView</class>
   <extends>QGraphicsView</extends>
   <header location="global">GeometryView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
